src/modules/tasks.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener('DOMContentLoaded', () => {
+    
+    const getItems = () => {
+        let items;
+
+        if(localStorage.getItem('items') === null){
+            items = [];
+        }else{
+            items = JSON.parse(localStorage.getItem('items'))
+        }
+
+        return items;
+    }
+
+      let items = getItems();
+    
+      items.sort((a, b) => a.index - b.index);
+  
+      document.querySelector('.container').innerHTML = `
+        <div class="title">
+          <h1>Today's to Do</h1>
+          <button type="button" class="reload"><i class="fa-solid fa-arrows-rotate"></i></button>
+        </div>
+        <form class="todo-form">
+          <input type="text" name="text" id="text-add" placeholder="Add to your list...">
+          <button type="submit" id="submit-btn"><i class="fa-solid fa-right-to-bracket"></i></button>
+        </form>
+        <ul class="list-container">
+        </ul>
+        <button type="button" id="delete-btn">Clear all completed</button>
+      `;
+  
+    const addItem = (item) => {
+      const listItems = document.querySelector('.list-container');
+  
+      const listItem = document.createElement('li');
+  
+      listItem.innerHTML = `
+          <div>
+            <input type="checkbox" name="checkbox" id="checkbox">
+            <input type="text" class="edit-input" id="${item.index}" name="task" value="${item.description}">
+          </div>
+          <button type="button" class="list-btn" ><i class="fa-solid fa-ellipsis-vertical"></i></button>
+          <button type="button" class="remove-btn" id="${item.index}"><i class="fa-solid fa-trash-can"></i></button>
+        `;
+  
+        listItems.insertAdjacentElement('beforeend', listItem);
+  
+        const editInput = document.querySelectorAll('.edit-input');
+        editInput.forEach((input) => {
+          input.addEventListener('input', ()=>{
+            const editId = parseInt(input.id, 10);
+            let item = JSON.parse(localStorage.getItem('items'));
+            items = item;
+            items[editId].description = document.getElementById(editId).value;
+            item = items;
+            localStorage.setItem('items', JSON.stringify(items));
+          })
+        })
+  
+    };
+
+    items.forEach((item) => addItem(item));
+  
+    document.querySelector('.todo-form').addEventListener('submit', (e) => {
+      e.preventDefault();
+      const inputValue = document.querySelector('#text-add').value;
+      const index = items.length;
```

Comment:
- [ ] Great job implementing the required functionalities. :+1: Let's work on some minor detail to achieve the project requirements. When a new item is added, its index should be the length of the new array. Currently, the index is being set to the length of the old array. Kindly work on this issue. 🤝️

**Project requirements**
![image](https://user-images.githubusercontent.com/17243325/222231068-0e03e52d-b1e8-40f0-af0d-918285e17787.png)

Reply:
- You have done well, but as the previous reviewer suggested, the index of the last task added should be equal to the length of the list. You are yet to correct this, please fix it 🙏. I suggest adding `1` to `item.length`. 
Your project:
<img width="978" alt="jaethzaeheheh" src="https://user-images.githubusercontent.com/73607512/222263731-db8e871d-2737-4a1f-95a0-bc134fa9e3fa.PNG">